frontend/packages/topology/src/components/quick-search/utils/quick-search-utils.tsx

```diff
@@ -3,15 +3,37 @@ import { useTranslation } from 'react-i18next';
 import { TextList, TextListItem } from '@patternfly/react-core';
 import { history, removeQueryArgument } from '@console/internal/components/utils';
 import { CatalogItem } from '@console/dynamic-plugin-sdk';
-import { keywordCompare } from '@console/dev-console/src/components/catalog/utils/catalog-utils';
 import { QuickStart } from '@console/app/src/components/quick-starts/utils/quick-start-types';
 import {
   QuickStartContext,
   QuickStartContextValues,
 } from '@console/app/src/components/quick-starts/utils/quick-start-context';
 
+const keywordListCompare = (keywords: string[], item: CatalogItem): boolean => {
```

Comment:
I think this should also be changed for the main dev catalog. Search experience for overall catalog should remain same.

Reply:
Updated `keywordCompare` function.